CHANGELOG.md

```diff
@@ -11,6 +11,52 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.12.0 - 2020-12-22
+
+Includes all the changes from [0.11.1].
+
+### Added
+
+- Add support for rule-level AppliedTo for Antrea-native policies. ([#1396](https://github.com/vmware-tanzu/antrea/pull/1396), [@Dyanngg])
+  * Ability to select different endpoints on which to apply the different rules within the same policy, without having to define multiple policies
+  * For a given policy, either the policy-level AppliedTo field must be used, or the rule-level AppliedTo fields
+- Add support for port ranges in the rules of Antrea-native policies. ([#1557](https://github.com/vmware-tanzu/antrea/pull/1557), [@GraysonWu])
+- Introduce the FlowAggregator, an [IPFIX mediator] implementation to collect, process and export flow records generated by the Antrea Agents. ([#1671](https://github.com/vmware-tanzu/antrea/pull/1671) [#1677](https://github.com/vmware-tanzu/antrea/pull/1677), [@srikartati] [@dreamtalen] [@zyiou])
+  * Built using the [go-ipfix] library
+  * Processed flow records exported by the FlowAggregator are not missing any K8s contextual information (e.g. source / destination Pod names)
```

Comment:
This sentence is not very easy to understand to me. Does it mean "processed flow records" "exported by FlowAggregator"? Could we remove "processed"?